Subject: [PATCH] Orion: register UART1 on QNAP TS-209 and TS-409

Register UART1 on QNAP TS-209 and TS-409 because the PIC controller is
connected to it.  This fixes a regression from 2.6.26.

Signed-off-by: Martin Michlmayr <tbm@cyrius.com>

--- a/arch/arm/mach-orion5x/ts209-setup.c	2008-08-13 07:39:42.000000000 +0000
+++ b/arch/arm/mach-orion5x/ts209-setup.c	2008-08-13 07:41:05.000000000 +0000
@@ -300,6 +300,7 @@
 	orion5x_i2c_init();
 	orion5x_sata_init(&qnap_ts209_sata_data);
 	orion5x_uart0_init();
+	orion5x_uart1_init();
 	orion5x_xor_init();
 
 	platform_device_register(&qnap_ts209_button_device);
--- a/arch/arm/mach-orion5x/ts409-setup.c	2008-08-13 07:39:55.000000000 +0000
+++ b/arch/arm/mach-orion5x/ts409-setup.c	2008-08-13 07:40:10.000000000 +0000
@@ -293,6 +293,7 @@
 	orion5x_eth_init(&qnap_tsx09_eth_data);
 	orion5x_i2c_init();
 	orion5x_uart0_init();
+	orion5x_uart1_init();
 
 	platform_device_register(&qnap_ts409_button_device);
 
